CMSgov/mint-app
# [EASI-3009] - Read only discussion updates

# EASI-3009

## Changes and Description

- Added role/roleDescription selection to discussions
- Implemented `mostRecentDiscussionRoleSelection` to populate recent role on discussion form
- Updated edit access to allow macs to comment on any model plan

Outstanding
Cypress/unit test updates

![Screenshot 2023-06-30 at 10 44 23 AM](https://github.com/CMSgov/mint-app/assets/95709965/84593356-18b8-424e-b4f5-ece86ed865dd)

## How to test this change

Log in as MAC and navigate to readonly discussion page
Comment and reply while selecting roles
Ensure recent roles are prepopulated

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
